src/__tests__/Routes.test.tsx

```diff
@@ -41,7 +41,7 @@ describe('src/Routes', () => {
             });
 
             expect(getLocation().pathname).toBe('/');
-            expect(document.body.firstChild).toBeEmptyDOMElement();
```

Comment:
this test is now showing the NotAuth page, would it make sense to adjust the test to show that or is this still meant to render nothing "/" (ignore the current change in there)

Reply:
I think the no access is fine

Reply:
Could you help re-work this test? It is failing because it shows the NotAuth page when running the test